### DISCUSSION TITLE
Use term translations in Search Query

### DISCUSSION THREAD

**Discussion by FloWaeger** [microsoft-search/pnp-modern-search#2102] [Q&A]
We've a taxonomy column which is mapped to the same term set as an user property and want to show only content which has the same terms tagged as the user has in his user profile so we did it like this:

{|RefinableString100:{user.userproperty}}

We are also using SP multi language and translated the terms of the mentioned term set.

Everything is fine, if the SiteCollection where the content is stored has the same default language as users current display language.

In this example we tagged the content with the term "Digitalisation" (German Translation: Digitalisierung). So the users with German display language get only content from SiteCollections with German default language because in this case the search stores "Digitalisierung" in RefinableString100 and Users with English display language get only content from SiteCollections with English default language because there is "Digitalisation" stored in RefinableString100.

Perhabs we can have the same functionality in the query as in the templates (autoRefinableString100)? Another way could be same functionality for user properties as for page properties (user.userproperty.TermID)?

Best regards
Florian

**Comment by wobba** [comment]
There is no easy way to solve this as the query variable will expand to the term label and not the id from the available API's.
